src/app/components/VerifyingIfAnswerIsCorrect.js

```diff
@@ -0,0 +1,6 @@
+
+export default function VerifyingAnswerIsCorrect(userAnswer, rightAnswer) {
```

Comment:
I think this name is overcomplicated. What you think about "AnswerVerification" or "IsAnswerCorrect"?

Reply:
You're right, the shorter name will be better and cleaner.